### PR TITLE
Fix Wooden Barricades not taking bullet damage

### LINKED ISSUE #13154: Lasers do no damage to wooden barricades.
What it says on the tin, I fired a bunch of lasers directly at one (clicking on it) and checked its health, didn't go down at all.

### PR DESCRIPTION
**Pull request by neersighted**
Fixes #13154 
It's a simple fix and we may want damage types to be scaled.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
:+1:

**Comment by Jordie0608** [comment]
You'll want to check for bullet damage type so a decloner radiation beam doesn't break wooden barricades.

**Comment by Zilenan91** [comment]
Can you also make the barricades take default melee damage because without the attack cooldown glitch on them these things were unbearably tanky.

**Comment by neersighted** [comment]
@Zilenan91 How's that?

**Comment by Zilenan91** [comment]
They take less damage from melee normally, so without the glitch where there is no attack cooldown on hitting them they took annoyingly long to break.

**Comment by KorPhaeron** [comment]
They're barricades they're meant to be tanky so they're worth building

**Comment by neersighted** [comment]
Should I revert the damage tweaks? It's a minor buff to brute, but mostly a buff to fire (which makes more sense).

File: code/game/machinery/deployable.dm [hunk 1]
```diff
@@ -95,12 +96,22 @@ obj/structure/barricade/wooden/proc/take_damage(damage, leave_debris=1, message)
...
+			if(BURN)
+				damage = W.force * 1.5
+			if(BRUTE)
 				damage = W.force * 1
-			if("brute")
-				damage = W.force * 0.75
 		take_damage(damage)
 
+/obj/structure/barricade/wooden/bullet_act(var/obj/item/projectile/P)
+	if(P)
```
Comment (phil235) [inline code/game/machinery/deployable.dm]: why check for P? is this something other bullet_act() procs do?

**Comment by Bawhoppen** [comment]
I remember jugs used to be unable to destroy these things making them OP in cult rounds, check on that and if its still the case maybe fix it too?

File: code/game/machinery/deployable.dm (hunk 1 above)
Comment (neersighted) [inline code/game/machinery/deployable.dm]: yes, other procs seem to check for P. I can remove it if needed.

**Pull request merged**